content/applications/hr/time_off/my_time.rst

```diff
@@ -0,0 +1,106 @@
+=======
+My time
+=======
+
+The :guilabel:`My Time` section of the *Time Off* application houses all the various time off
+information for the signed-in user.
+
+This includes the main *Time Off* dashboard, which displays an overview of the user's various time
+off balances, as well as the user's time-off requests and allocations.
+
+.. _time_off/dashboard:
+
+Dashboard
+---------
+
+All users have access to the time off dashboard, which is the default view in the *Time Off*
+application. The dashboard can also be accessed at any point in the application by navigating to
+:menuselection:`Time Off app --> My Time --> Dashboard`.
+
+The current year is displayed, and the current day is highlighted in red.
+
+To change the view, click on the :guilabel:`Year` button to reveal a drop-down menu. Then, select
+either :guilabel:`Day`, :guilabel:`Week`, or :guilabel:`Month` to present the calendar in that
+corresponding view.
+
+To change the displayed dates, click the :icon:`fa-arrow-left` :guilabel:`(left arrow)` or
+:icon:`fa-arrow-right` :guilabel:`(right arrow)` icons to the left of the :guilabel:`Year` button.
+The calendar view adjusts in increments of the selected view.
+
+For example, if :guilabel:`Year` is selected, the arrows adjust the view by one year.
+
+To reset the view so it includes the current date, click the :guilabel:`Today` button.
+
+Above the calendar view is a summary of the user's time off balances. Every time off type that has
+been allocated appears in its own summary box. Each summary lists the type of time off, the
+corresponding icon, the current available balance (in hours or days), and an expiration date (if
+applicable).
+
+To view the full details of a time off balance, click the :icon:`fa-question-circle-o`
+:guilabel:`(question mark)` icon at the end of the :guilabel:`(DAYS/HOURS) AVAILABLE` on the time
+off summary. The complete details are presented in a popover window, including the
+:guilabel:`Allocated` time, future :guilabel:`Accrual` time, :guilabel:`Approved` time off
+scheduled, :guilabel:`Planned` time off, and the currently :guilabel:`Available` time off.
+
+.. image:: my_time/balance-details.png
+   :align: center
+   :alt: A view of the complete time off balance details in the popover window.
+
+A user can also select a future date to see an estimate of how much time they should accrue by that
+ point. On the right side of the time off summary blocks, there is a :guilabel:`Balance at the
+ (date)` field. Click on the date, and a calendar selector popover appears.
```

Comment:
```suggestion
A user can also select a future date to see an estimate of how much time they should accrue by that
point. On the right side of the time off summary blocks, there is a :guilabel:`Balance at the (date)` field.
Click on the date, and a calendar selector popover appears.
```
Can you please clarify a bit more where this specific element can be found on the dashboard? Because I am having a difficult time locating it in the runbot I am using for this review. Perhaps I am just not looking hard enough? 😆 

![Screenshot 2024-06-21 093623](https://github.com/odoo/documentation/assets/73958186/f62148fe-7fca-4731-9dd5-8cf29746e746)

Reply:
Oh!! You aren't seeing this because in the runbot you do not have any time off assigned to you that accrues over time. You have only time that is assigned and granted immediately, with no waiting. So this is only visible when you can actually "earn" or accrue the time over a period of time. I'll add that note!